Mathlib/Data/List/MinMax.lean

```diff
@@ -530,6 +530,17 @@ theorem min_le_of_le (l : List α) (a : α) {x : α} (hx : x ∈ l) (h : x ≤ a
 
 end OrderTop
 
+/-- If `a ≤ x` for some `x` in the list `l`, and `b : α`, then `a ≤ l.foldr max b`. -/
+theorem le_max_of_le' {l : List α} {a x : α} (b : α) (hx : x ∈ l) (h : a ≤ x) :
```

Comment:
Could you please also provide the dual `min_le_of_le'`?

Reply:
Done!